### ISSUE TITLE
Warning with ngc compiler

### ISSUE THREAD

**Issue by tsm91** [fknop/angular-pipes#27]
Hi, with the new major version i got this warning message popping up if using the ngc compiler.

```sh
WARNING in ./~/angular-pipes/src/index.js
Cannot find SourceMap 'index.js.map': Error: Can't resolve './index.js.map' in '/home/tsm/Sites/deploy-fe/node_modules/angular-pipes/src'
 @ ./src/app/shared/modules/shared.module.ts 9:22-46
 @ ./src/compiled/src/app/app.module.ngfactory.ts
 @ ./src/main.browser.aot.ts
```

version used: "angular-pipes": "^5.0.9",

**Comment by fknop** [comment]
That's weird, I just installed the package I have the `.map` file in the src folder. Could you paste me your directory structure / files under `node_modules/angular-pipes` ?

**Comment by fknop** [comment]
Ok, nevermind, it looks like `js` files are missing. I'll look into it.

**Comment by fknop** [comment]
I just published `5.0.10`. It should work.

**Comment by tsm91** [comment]
it does, thx
